tailwind.config.ts

```diff
@@ -94,6 +94,76 @@ export default {
       'link': 'oklch(var(--link) / <alpha-value>)',
 
       'focus': 'oklch(var(--focus) / <alpha-value>)',
+
+      // NC colors
+      'neon-coral': {
+        DEFAULT: 'oklch(var(--neon-coral) / <alpha-value>)',
+        dark: 'oklch(var(--neon-coral--dark) / <alpha-value>)',
+      },
+      'sea-green': {
+        DEFAULT: 'oklch(var(--sea-green) / <alpha-value>)',
+        dark: 'oklch(var(--sea-green--dark) / <alpha-value>)',
+      },
+      'slate-blue': {
+        DEFAULT: 'oklch(var(--slate-blue) / <alpha-value>)',
+        dark: 'oklch(var(--slate-blue--dark) / <alpha-value>)',
+      },
+      'navy-taupe': {
+        DEFAULT: 'oklch(var(--navy-taupe) / <alpha-value>)',
+        dark: 'oklch(var(--navy-taupe--dark) / <alpha-value>)',
+      },
+      'cyber-grape': {
+        DEFAULT: 'oklch(var(--cyber-grape) / <alpha-value>)',
+        dark: 'oklch(var(--cyber-grape--dark) / <alpha-value>)',
+      },
+      'mustard': {
+        DEFAULT: 'oklch(var(--mustard) / <alpha-value>)',
+        dark: 'oklch(var(--mustard--dark) / <alpha-value>)',
+      },
+      'rich-black': {
+        DEFAULT: 'oklch(var(--rich-black) / <alpha-value>)',
+        dark: 'oklch(var(--rich-black--dark) / <alpha-value>)',
+      },
+      'charcoal': {
+        DEFAULT: 'oklch(var(--charcoal) / <alpha-value>)',
+        dark: 'oklch(var(--charcoal--dark) / <alpha-value>)',
+      },
+      'platinum': {
+        DEFAULT: 'oklch(var(--platinum) / <alpha-value>)',
+        dark: 'oklch(var(--platinum--dark) / <alpha-value>)',
+      },
+      'sea-serpent': {
+        DEFAULT: 'oklch(var(--sea-serpent) / <alpha-value>)',
+        dark: 'oklch(var(--sea-serpent--dark) / <alpha-value>)',
+      },
+      'purple-pizazz': {
+        DEFAULT: 'oklch(var(--purple-pizazz) / <alpha-value>)',
+        dark: 'oklch(var(--purple-pizazz--dark) / <alpha-value>)',
+      },
+      'paradise-pink': {
+        DEFAULT: 'oklch(var(--paradise-pink) / <alpha-value>)',
+        dark: 'oklch(var(--paradise-pink--dark) / <alpha-value>)',
+      },
+      'cerulean-blue': {
+        DEFAULT: 'oklch(var(--cerulean-blue) / <alpha-value>)',
+        dark: 'oklch(var(--cerulean-blue--dark) / <alpha-value>)',
+      },
+      'kiwi': {
+        DEFAULT: 'oklch(var(--kiwi) / <alpha-value>)',
+        dark: 'oklch(var(--kiwi--dark) / <alpha-value>)',
+      },
+      'neon-carrot': {
+        DEFAULT: 'oklch(var(--neon-carrot) / <alpha-value>)',
+        dark: 'oklch(var(--neon-carrot--dark) / <alpha-value>)',
+      },
+      'barbie-pink': {
+        DEFAULT: 'oklch(var(--barbie-pink) / <alpha-value>)',
+        dark: 'oklch(var(--barbie-pink--dark) / <alpha-value>)',
+      },
+      'tomato': {
+        DEFAULT: 'oklch(var(--tomato) / <alpha-value>)',
+        dark: 'oklch(var(--tomato--dark) / <alpha-value>)',
+      },
```

Comment:
Ah, so we specifically _don't_ want to do this. We only want to include semantic tokens in the theme, and then we want to have the theme specify the colors. This will allow users to dynamically provide/respecify colors.

Reply:
implemented here [2e91f01](https://github.com/complexdatacollective/Studio/pull/45/commits/2e91f014beba18891401648060ac4518146717d9)